sections/pickup-availability.liquid

```diff
@@ -1,4 +1,6 @@
-{% comment %}theme-check-disable UndefinedObject{% endcomment %}
+{% comment %}
+  theme-check-disable UndefinedObject
+{% endcomment %}
```

Comment:
Probably don't need to break unconditionally. Unless it's breaking because that's more than 100 chars? 🤔